.circleci/config.yml

```diff
@@ -419,38 +401,25 @@ workflows:
             parameters:
               # test the lowest numpy version and the highest minor of each
               # currently deployed
-              numpy-version: [==1.17.3, ~=1.18.0, ~=1.19.0, ~=1.20.0, ~=1.21.0]
+              numpy-version: [==1.19.1, ~=1.20.0, ~=1.21.0, ~=1.22.0]
```

Comment:
Perhaps include `~=1.19.0` as well?

Reply:
My reasoning is that NumPy is pretty trustworthy to not break backwards compatibility between 1.19.1 and ~=1.19.0 and it saves a bit of solve time not to test.